### PR TITLE
partially solves #176 - prepared majority of the apps for store listing

### PR DESCRIPTION
**Pull request by djmgit**
partially fixes issue #176. Prepared the following apps for storelisting.

```
dcuphanduy
movietweets
yasoob
fossasia-histogram
yathannsh
KnowTheDiff
userprofile
LQL
webtweets
tweetsleaderboard
bolerplate
collegeElections
social-wall
querybrowser
superwomen7
histogram
sentimentVisualizer
oneClickDeploy
tweetfeed

```
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
@kavithaenair, @SKrPl , @Achint08 open for review

**Comment by djmgit** [comment]
Prepared majority of the apps for store listing. We can iterate from here as the PR has already become long.

File: fossasia-histogram/others.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This a.pp uses morris.js, a simple light weight library for plotting
```
Comment (kavithaenair) [inline fossasia-histogram/others.md]: Correct the line here.

File: histogram/getStarted.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+This application visualises loklak aggregation data using bar graphs.
+
+Enter query in the input field and enter start and ebd dates
```
Comment (kavithaenair) [inline histogram/getStarted.md]: Typo here

File: knowTheDiff/getStarted.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+This is an application to show diffence between loklak **search api**
```
Comment (kavithaenair) [inline knowTheDiff/getStarted.md]: Typo

File: knowTheDiff/getStarted.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+and loklak **aggregation api**. There is an in-app tutoral to use the
```
Comment (kavithaenair) [inline knowTheDiff/getStarted.md]: Typo

File: superwoman7/getStarted.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+This app recreates **tweet timeline**.
+
+Just enter a tweet word or query in the input field
+and you wll be presented with a timeline with various
```
Comment (kavithaenair) [inline superwoman7/getStarted.md]: Typo

File: tweetfeed/appUse.md [hunk 6]
```diff
@@ -0,0 +1 @@
+This app can be used to visualise tweedfedd related to your query.
```
Comment (kavithaenair) [inline tweetfeed/appUse.md]: Typo

File: webtweets/getStarted.md [hunk 7]
```diff
@@ -0,0 +1,2 @@
+An Application to show tweets about FOSSASIA. The app shows tweets
+onr after the other in an automated way.
```
Comment (kavithaenair) [inline webtweets/getStarted.md]: Typo

File: boilerplate/getStarted.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+This application can be used as a template to get started with
+**loklak app development** quickly.
+Just create a copy of this app, rename this app, configure the
+app.json and edit the files to create your own app. This app
+serves as an alternative of usng loklakinit tool or creating an
```
Comment (Achint08) [inline boilerplate/getStarted.md]: Typo

**Comment by djmgit** [comment]
@daminisatya 
Test link : http://loklakapps.surge.sh/

click on any of the apps listed above

**Pull request merged**